ra_aid/__main__.py

```
@@ -1068,13 +1052,13 @@ def main():
 
                 # Process initial task if provided
                 if not start_interactive:
-                    process_task(args)
+                    process_task(args, mcp_use_client_instance)
```

Ruff (0.8.2)

Undefined name `mcp_use_client_instance`

(F821)